e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -39,90 +38,9 @@ describe('User Admin', () => {
     const nameColIndex = await adminTable.getNameColindex();
 
     //click on the name link to navigate to the admin-user-profile page
-    const userProfileInfo = await userAdminPage.clickNameLink(1, nameColIndex);
+    const userProfileAdmin = await userAdminPage.clickNameLink(1, nameColIndex);
     //navigate to admin-user-profile page
-    await userProfileInfo.waitForLoad();
-  });
-
-  test('Verify admin-user-profile page ui, update initialCredits and update institution', async () => {
```

Comment:
no longer valid - new test which covers this (and the new page) is user-profile-admin.spec